src/riak_kv_quanta.erl

```diff
@@ -0,0 +1,250 @@
+-module(riak_kv_quanta).
+
+-export([quanta/3,
+         timestamp_to_ms/1,
+         ms_to_timestamp/1]).
+
+%% @doc This module serves to generate time quanta on multi - (year, month, day, hour, minute,
```

Comment:
Sorry to be pedantic - this needs to be above the module declaration. Also, you should probably verify that the type docs come out as expected, I've found them to be fiddly in the past (such that I don't remember how to do them correctly).